Cargo.toml

```diff
@@ -13,7 +13,10 @@ license = "Apache-2.0"
 
 [features]
 default = [ "cluster-context" ]
-cluster-context = [ "dep:k8s-openapi" ]
+cluster-context = [ "k8s-openapi" ]
+
+[package.metadata.docs.rs]
+features = ["k8s-openapi/v1_26"]
```

Comment:
should we bump the k8s-openapi dev-dependency to consume the same feature version?

Reply:
no, this isn't really needed. The k8s version just enables/removes k8s resource types. This isn't needed by the tests